coordinator/clients/prover-client/file-based-client/src/main/kotlin/net/consensys/zkevm/coordinator/clients/prover/FileBasedExecutionProverClientV2.kt

```diff
@@ -33,48 +26,6 @@ data class BatchExecutionProofRequestDto(
   val blocksData: List<RlpBridgeLogsData>
 )
 
-internal class ExecutionProofRequestDataDecorator(
```

Comment:
This class is deleted as I would like to move the responsibility of retrieving "L2 state root hash and bridge logs" from execution prover client to the execution proof coordinator, and hence the prover client would just be responsible for creating the proof request based on the data passed by the caller.